src/kouros/pos/pos_manager.go

```diff
@@ -23,10 +23,102 @@ type POSManager interface {
 	// ListDevices method lists the devices in PoseidonOS
 	// The function does not require any parameters
 	// A successful ListDevices call returns response in JSON format
-	ListDevices() (response *pb.ListDeviceResponse, err error)
+	ListDevices() (*pb.ListDeviceResponse, error)
 
 	// Create Array method creates an array in PoseidonOS
 	// The function takes a command in JSON format as the parameter
 	// A successful Create call returns response in JSON format
-	CreateArray(param pb.CreateArrayRequest_Param) (response *p.CreateArrayResponse, err error)
+	CreateArray(param *pb.CreateArrayRequest_Param) (*pb.CreateArrayResponse, error)
+
+	// Add device command add spare device in PoseidonOS Array
+	// The function takes a byte array in JSON format as parameter and returns response in JSON format
```

Comment:
Please change the comment related to JSON format.

Reply:
I will correct the description